Flow.Launcher.Infrastructure/Image/ImageCache.cs

```diff
@@ -3,97 +3,81 @@
 using System.Collections.Generic;
 using System.Linq;
 using System.Threading;
+using System.Threading.Tasks;
 using System.Windows.Media;
-using FastCache;
-using FastCache.Services;
+using BitFaster.Caching.Lfu;
 
 namespace Flow.Launcher.Infrastructure.Image
 {
-    public class ImageUsage
-    {
-        public int usage;
-        public ImageSource imageSource;
-
-        public ImageUsage(int usage, ImageSource image)
-        {
-            this.usage = usage;
-            imageSource = image;
-        }
-    }
-
     public class ImageCache
     {
-        private const int MaxCached = 150;
+        private const int MaxCached = 10;
```

Comment:
is it enough for explorer plugin that has many results?

Reply:
Oops sorry I forget to change it back. It's for testing🥲